demos/coreMQTT/mqtt_demo_mutual_auth.c

```diff
@@ -779,6 +779,8 @@ static BaseType_t prvConnectToServerWithBackoffRetries( NetworkContext_t * pxNet
         }
     } while( ( xNetworkStatus != TRANSPORT_SOCKET_STATUS_SUCCESS ) && ( xBackoffStatus == pdPASS ) );
 
+    xStatus = ( xBackoffStatus == pdPASS ) ? pdPASS : pdFAIL;
```

Comment:
Can we just return `xBackoffStatus` and remove `xStatus` altogether?

Reply:
Oh I think the issue is that `xBackoffStatus` is using `pdFALSE` instead of `pdFAIL`. Although, I suppose they do map to the same most value for the most part.

Reply:
xStatus removed 👍